lib/datasource/types.ts

```diff
@@ -54,6 +55,7 @@ export interface ReleaseResult {
   homepage?: string;
   sourceUrl?: string;
   sourceDirectory?: string;
+  registryUrl?: string;
```

Comment:
Maybe only add to release?

Reply:
The result will be cascaded later anyway. I prefer not to put a registry per release for all the ones which don't merge